app/src/main/res/layout/dialog_image_description.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingBottom="0dp">
+
+    <com.github.chrisbanes.photoview.PhotoView
+        android:id="@+id/imageDescriptionView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:contentDescription="@string/post_media_image"/>
+
+    <com.google.android.material.textfield.TextInputLayout
+        style="@style/TuskyTextInput"
+        android:id="@+id/imageDescriptionTextLayout"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:layout_marginStart="?dialogPreferredPadding"
+        android:layout_marginTop="?dialogPreferredPadding"
+        android:layout_marginEnd="?dialogPreferredPadding"
+        app:hintEnabled="false"
```

Comment:
This uses the hint on the actual text field.

The one on the InputLayout would be slicker but our text is much too long for that.
(It would be cut when floating - only one line shown - and it would still be cut when moved to the border when the input field is focused.)